pystac/rel_type.py

```diff
@@ -0,0 +1,33 @@
+from enum import Enum
+
+
+class RelType(str, Enum):
+    """A list of common rel types that can be used in STAC Link metadata."""
```

Comment:
Perhaps we should link to the best practices docs here so users who want to learn more about how to use the reltypes have an easy find?

Reply:
Yeah, that's a good idea. I'll add a reference to those docs.